mix.exs

```diff
@@ -35,7 +35,9 @@ defmodule RemotePersistentTerm.MixProject do
       {:ex_aws_s3, "~> 2.0"},
       {:configparser_ex, "~> 4.0", optional: true},
       {:hackney, "~> 1.9"},
-      {:sweet_xml, "~> 0.6"}
+      {:sweet_xml, "~> 0.6"},
+      {:mock, "~> 0.3.0", only: :test},
+      {:mint, "~> 1.0", only: :test}
```

Comment:
Wasn't sure whether it was best to add Mint as a test dep to use the struct, or to create a struct that mimics it locally in our test folder to avoid adding another dependency. @sneako wdyt?